File: ALGORITMO E PROGRAMAÇÃO/LISTA 06/Exe01.js
const entrada = require('prompt-sync')({
    sigint: true
})

function principal(){
    const tamanho_vetor = Number(entrada('Informe o tamanho do vetor: '))
    let vetor_a = Array(tamanho_vetor)
    let vetor_b = Array(tamanho_vetor)

   for(let i = 0; i < tamanho_vetor; i++){
        vetor_a[i] = Number(entrada(`Informe o elemento ${i} do vetor: `))
       
    }
    let j = 0
    for(let i = vetor_b.length - 1; i >= 0; i--){
        vetor_b[i] = vetor_a[j]
        j++
    }
   console.log(vetor_a)
    console.log(vetor_b)
    
    

}

principal()

File: ESTRUTURA DE DADOS/LISTA03/Exe01.c
/*Leia um vetor A com N elementos e escreva um vetor B, com os mesmos elementos de A,
sendo que estes deverão estar invertidos, ou seja, o 1o elemento de A deve ser o último
elemento de B; o 2o elemento de A deve ser o penúltimo elemento de B e assim por diante.*/
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main() {
    int tamanho, i, j;
    printf("Informe o tamanho do vetor: ");
    scanf("%d", &tamanho);
    int VetorA[tamanho], VetorB[tamanho];
    for(i = 0; i < tamanho; i++) {
        printf("Digite o elemento %d: ",i);
        scanf("%d", &VetorA[i]);
    }
      printf("O vetorB é [");
    for(i=0, j = tamanho - 1; i<tamanho && j >= 0; i++, j-- ) {
        VetorB[i] = VetorA[j];
        printf(" %d ", VetorB[i]);
    }
    printf("]");
    return 0;
}
